src/main/java/com/enderio/base/common/init/EIOItems.java

```diff
@@ -259,9 +259,9 @@ private static RegiliteItem<FixedCapacitorItem> fixedCapacitor(String name, ICap
     public static final RegiliteItem<BrokenSpawnerItem> BROKEN_SPAWNER = ITEM_REGISTRY
         .registerItem("broken_spawner", BrokenSpawnerItem::new)
         .setModelProvider(EIOModel::fakeBlockModel)
+        .addItemTags(EIOTags.Items.STORED_ENTITY)
         .setTab(EIOCreativeTabs.MAIN)
         .setTab(EIOCreativeTabs.SOULS, modifier -> modifier.acceptAll(BrokenSpawnerItem.gePossibleStacks()));
```

Comment:
Can you change this spelling mistake just in case? `BrokenSpawnerItem.gePossibleStacks()`